packages/sdk/src/shared/index.ts

```
@@ -0,0 +1,844 @@
+
+  public async componentReloadProps(opts: ComponentReloadPropsOpts) {
+    // RpcActionReloadPropsInput
+    const body = {
+      async_handle: this.asyncResponseManager.createAsyncHandle(),
+      external_user_id: opts.userId,
+      id: opts.componentId,
+      configured_props: opts.configuredProps,
+      dynamic_props_id: opts.dynamicPropsId,
+      environment: this.environment,
+    };
+    return await this.makeConnectRequestAsync<{}>("/components/props", {
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)